osquery/tables/system/windows/programs.cpp

```diff
@@ -88,6 +181,130 @@ void keyEnumPrograms(const std::string& key,
   }
 }
 
+void genMsixPrograms(const std::string& key, QueryData& results) {
+  std::set<std::string> packageFamilyNameProcessed;
+  QueryData regResults;
+  queryKey(key, regResults);
+  for (const auto& rKey : regResults) {
+    // Each subkey represents a package, skip if not a subkey
+    if (rKey.at("type") != "subkey") {
+      continue;
+    }
+    const auto& regPath = rKey.at("path");
+    const auto& regPackageFullName = rKey.at("name");
+
+    // Get all registry entires for the package
+    QueryData appResults;
+    queryKey(regPath, appResults);
+    Row result;
+
+    result["identifying_number"] =
+        identifyingNumberFromPackageFullName(regPackageFullName);
+
+    for (const auto& aKey : appResults) {
+      auto name = aKey.find("name");
+
+      if (name->second == "PackageRootFolder") {
+        result["install_location"] = aKey.at("data");
+        std::string filePath =
+            result["install_location"] + "\\AppxManifest.xml";
+
+        std::ifstream xmlFile(filePath);
+        if (!xmlFile) {
+          // Skip this program, as we cannot find the manifest file
+          // We can extract some information from the registry key
+          // <PackageName>_<Version>_<Architecture>__<PublisherHash>
+          // Example:
+          // MSTeams_25060.205.3499.6849_arm64__8wekyb3d8bbwe
+          // But all proper MSIX packages have an AppxManifest.xml file
+          LOG(INFO) << "Cannot find manifest file:'" + regPath + "'";
```

Comment:
How often will this happen? I'm wondering if it's better as verbose

Reply:
It happened twice on my clean windows vm:
<img width="1275" alt="Screenshot 2025-03-29 at 8 45 50 AM" src="https://github.com/user-attachments/assets/5767c815-b6a7-46ad-9ef9-f4f5a81ead16" />

happy to switch to `LOG(VERBOSE)` to quiet down the logs.